addons/crm/models/crm_stage.py

```diff
@@ -37,9 +37,17 @@ def default_get(self, fields):
     probability = fields.Float('Probability (%)', required=True, default=10.0, help="This percentage depicts the default/average probability of the Case for this stage to be a success")
     on_change = fields.Boolean('Change Probability Automatically', help="Setting this stage will change the probability automatically on the opportunity.")
     requirements = fields.Text('Requirements', help="Enter here the internal requirements for this stage (ex: Offer sent to customer). It will appear as a tooltip over the stage's name.")
-    team_id = fields.Many2one('crm.team', string='Team', ondelete='set null',
+    team_id = fields.Many2one('crm.team', string='Sales Channel', ondelete='set null',
         help='Specific team that uses this stage. Other teams will not be able to see or use this stage.')
     legend_priority = fields.Text('Priority Management Explanation', translate=True,
         help='Explanation text to help users using the star and priority mechanism on stages or issues that are in this stage.')
     fold = fields.Boolean('Folded in Pipeline',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
+
+    #This field for interface only
+    number_of_teams = fields.Integer('number_of_teams', compute='_compute_number_of_teams')
```

Comment:
I don't understand that field. Shouldn't you compute the number of teams using this stage ? Either there is a team_id set and it is 1, either team_id is null and this is the number of active teams.

I see it is used to hide team_id. What is the use case ?

I would also probably name it team_count, to match the team_id field.